frontend/src/pages/DocumentPage.tsx

```diff
@@ -1,10 +1,11 @@
+import { RouteComponentProps } from 'wouter';
 import TranscriptionEditor from '../editor/TranscriptionEditor';
 
-export default function DocumentPage() {
+export default function DocumentPage({ params }: RouteComponentProps<{ documentId: string }>) {
```

Comment:
Now that we have the gallery which serves as a "home page" I would wish for a link from the document page back to the gallery